src/index.ts

```diff
@@ -43,6 +43,26 @@ const ts2gas = (source: string, transpileOptions: ts.TranspileOptions = {}) => {
       return (file: ts.SourceFile) => file;
     };
 
+    const restoreImportIdBuilder = (kind: ts.SyntaxKind, nodeFilter: NodeFilter): TransformerFactory =>
```

Comment:
Add a comment as to what this does. Like the other functions.
Be sure to include `@param ...`

Reply:
@grant Will do after commute. When at home. Current ts2gas npm package is buggy and thus breaks clasp. Plus you need to add a tsc build when publishing.